binding.gyp

```diff
@@ -48,6 +48,27 @@
             "BRUTE_FORCE"
           ]
         }],
+        # Depending on version of Python, IBM i can identify as AIX or OS400
+        ['OS=="aix" or OS=="os400"', {
+          "sources": [
+            "src/watchman/BSER.cc",
+            "src/watchman/WatchmanBackend.cc",
```

Comment:
Apologies for the slow response on this PR.

One problem is that I don't think watchman is supported on AIX or OS400. There is also no inotify support as far as I can tell (I don't have access to a system to test this). Without either of those, this module isn't very useful (can't actually watch files). Do you know if these OSes have another file watching mechanism?

I'm also somewhat concerned about adding these as I have no way of testing or providing precompiled builds for these platforms. Do you have access to such a system to test with?

Reply:
Apologies for the late response.

Watchman isn't supported, but I didn't disable it, because Watchman is an external tool - I was thinking that it might support it or an alternative with a compatible interface might. It might be a good idea to disable it though.

It's using the "legacy" backend which is brute force; AFAIK there is no watching API, but I might be wrong. I can ask my contacts at IBM.

I understand the concerns about not being able to test things. I do have access to a system for testing. FWIW not having a binary build is OK; the user I'm doing this for is already used to having the dev toolchain to build native parts already.

(FWIW, the motivation to upstream this was things depending on watcher being *extremely* common in some ecosystems. Finding and patching every dependency to pull a patched copy was too onerous.)

Reply:
Putting a ping out here - would like to know if the above is still blocking.